src/shared/components/Table/Table.module.css

```
@@ -0,0 +1,220 @@
+	white-space: nowrap;
+	vertical-align: middle;
+	user-select: none;
+	position: relative;
+}
+
+.headerContent {
+	width: 100%;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+	word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)